lib/wrapInitialConfig.js

```diff
@@ -10,6 +10,15 @@ module.exports = (api) => {
     'hash-module-ids',
   ];
 
+  if (options && options.allowedPlugins) {
```

Comment:
This block looks too complicated. I don't think you need this at all.

Reply:
Yeah something like `options.allowedPlugins ||= []`

Reply:
What about to describe default options object and merge it with `Object.assign` or with `webpack-merge`, which is dependency for `@vue/cli-service`?

Reply:
We can do that in index.js. Let's use `Object.assign`